test/extra/NotificationTest.cpp

```diff
@@ -83,7 +83,6 @@ TEST_F(NotificationTest, BasicNotificationTestWithPayload) {
 	// Create a second sink with a different source filter to verify messages
 	// arrive at the right sink
 	auto source_filter2 = getUUri(0x8001);
-	auto on_rx2 = [](const v1::UMessage& message) { FAIL(); };
 	auto maybe_sink2 = communication::NotificationSink::create(
 	    transport, std::move(on_rx), std::move(source_filter2));
```

Comment:
Actually this is a bug in the test -- the call to NotificationSink::create on the next line should use the on_rx2 call back instead of the on_rx callback.